### DISCUSSION TITLE
Validation of ZUGFeRD PDF/A-3a brings up "Not a PDF/A-3" result

### DISCUSSION THREAD

**Discussion by svenlohberg** [ZUGFeRD/mustangproject#522] [Q&A]
I am using the PDF XPansion SDK to generate ZUGFeRD PDF documents. These are created as PDF/A-3a documents. A validation of these files always brings up the message "Not a PDF/A-3" and the files are marked as invalid.

Shouldn't all PDF/A-3 files be valid independent from the used flavor a, b or u (PDF/A-3a, PDF/A-3b, PDF/A-3u)?

Unfortunately the SDK does not seem to allow creating PDF/A-3 documents without any flavor.

**Comment by jstaerk** [comment]
Hello
```
I am using the PDF XPansion SDK to generate ZUGFeRD PDF documents. These are created as PDF/A-3a documents. A validation of these files always brings up the message "Not a PDF/A-3" and the files are marked as invalid.

Shouldn't all PDF/A-3 files be valid independent from the used flavor a, b or u (PDF/A-3a, PDF/A-3b, PDF/A-3u)?
```
They are. And AFAIK Mustang does not restrict to a, b or u. For the PDF validation mustang uses verapdf, have you checked e.g. with verapdf.org itself if the document is proper PDF/A? 

kind regards
Jochen
